<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-MDMDGLO" failures="1" tests="5" name="EdgeTest" time="100.526" errors="0" timestamp="2024-03-20T00:05:52 EET">
  <testcase classname="testcase.MyFirstTest" name="Test" time="12.472"/>
  <testcase classname="testcase.MyFirstTest" name="Test" time="8.181"/>
  <testcase classname="testcase.MyFirstTest" name="Test" time="10.007"/>
  <testcase classname="testcase.MyFirstTest" name="Test" time="9.173"/>
  <testcase classname="testcase.MyFirstTest" name="Test" time="7.362">
    <failure type="java.lang.AssertionError" message="expected [10] but found [9]">
      <![CDATA[java.lang.AssertionError: expected [10] but found [9]
at testcase.MyFirstTest.Test(MyFirstTest.java:68)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Test -->
</testsuite> <!-- EdgeTest -->
